webapp/src/ts/effects/contacts.effects.ts

```diff
@@ -116,50 +121,47 @@ export class ContactsEffects {
     );
   },{ dispatch: false });
 
-  updateSelectedContactSummary = createEffect(() => {
+  receiveSelectedContactTargetDoc = createEffect(() => {
     return this.actions$.pipe(
-      ofType(ContactActionList.receiveSelectedContactReports),
-      withLatestFrom(
-        this.store.pipe(select(Selectors.getSelectedContact))
-      ),
-      exhaustMap(([, selectedContact]) => {
-        const selected: any = { ...selectedContact };
-        this.contactsActions.setContactsLoadingSummary(true);
-        this.tasksForContactService
-          .get(selected)
-          .then((tasks) => {
-            return this.contactsActions.updateSelectedContactsTasks(tasks);
-          });
-        return from(this.contactSummaryService.get(selected.doc, selected.reports, selected.lineage)).pipe(
-          map(summary => {
-            this.contactsActions.setContactsLoadingSummary(false);
-            return this.contactsActions.updateSelectedContactSummary(summary);
+      ofType(ContactActionList.loadSelectedContactTargetDoc),
+      exhaustMap(({ payload: { selected } }) => {
```

Comment:
This action doesn't need a payload, and can get the selectedContact directly from the state: 
`withLatestFrom(this.store.pipe(select(Selectors.getSelectedContact))`